src/internal/v1api/providers.go

```diff
@@ -34,12 +35,14 @@ func NewProviderGenerator(p provider.Provider, destination string) (ProviderGene
 
 // VersionListingPath returns the path to the provider version listing file.
 func (p ProviderGenerator) VersionListingPath() string {
-	return filepath.Join(p.Destination, "v1", "providers", p.Provider.Namespace, p.Provider.ProviderName, "versions")
+	listingPath := filepath.Join(p.Destination, "v1", "providers", p.Provider.Namespace, p.Provider.ProviderName, "versions")
```

Comment:
BUG: if the p.Destination has upper/lower cases it will break

Reply:
Fixed, thanks